MekHQ/src/mekhq/campaign/rating/AbstractUnitRating.java

```diff
@@ -413,6 +413,8 @@ protected BigDecimal getNumberUnits() {
      */
     protected abstract int calculateUnitRatingScore();
 
+    public abstract UnitRatingMethod getUnitRatingMethod();

```

Comment:
Since you have this in the interface, you shouldn't put it in the abstract unless you're adding logic to it.